pkgs/applications/misc/chrysalis/default.nix

```diff
@@ -1,35 +1,42 @@
 { lib, appimageTools, fetchurl }:
 
 let
+  owner = "keyboardio";
   pname = "chrysalis";
-  version = "0.12.0";
-in appimageTools.wrapAppImage rec {
+  version = "0.13.2";
   name = "${pname}-${version}-binary";
-
-  src = appimageTools.extract {
-    inherit name;
-    src = fetchurl {
-      url = "https://github.com/keyboardio/${pname}/releases/download/v${version}/${pname}-${version}.AppImage";
-      sha256 = "sha256-sQoEO1UII4Gbp7UbHCCyejsd94lkBbi93TH325EamFc=";
-    };
+  src = fetchurl {
+    url =
+      "https://github.com/${owner}/${pname}/releases/download/v${version}/${pname}-${version}-x64.AppImage";
+    sha512 =
+      "WuItdQ/hDxbZZ3zulHI74NUkuYfesV/31rA1gPakCFgX2hpPrmKzwUez2vqt4N5qrGyphrR0bcelUatGZhOn5A==";
   };
+  appimageContents = appimageTools.extract { inherit name src; };
+in appimageTools.wrapType2 rec {
```

Comment:
please use `finalAttrs` instead of `rec` to allow proper overriding (see https://nixos.org/manual/nixpkgs/stable/#mkderivation-recursive-attributes)

Reply:
ignoring.